book/src/repair-service.md

```diff
@@ -0,0 +1,47 @@
+# Repair Service
+
+The Repair Service is in charge of retrieving missing blobs that failed to be delivered by primary communication protocols like Avalanche. It is also the primary mechanism by which new nodes joining the network fetch old portions of the ledger.
```

Comment:
The book distinguishes the term "cluster" and "network". Can you too? The cluster is the set of nodes in gossip, whereas the network is how the nodes communicate. So, for example, we'd say "the cluster has 100 nodes running on a gigabit network."